test/images/agnhost/crd-conversion-webhook/converter/converter_test.go

```diff
@@ -92,7 +92,7 @@ request:
 				return
 			}
 
-			yamlSerializer := json.NewYAMLSerializer(json.DefaultMetaFactory, scheme, scheme)
+			yamlSerializer := json.NewSerializerWithOptions(json.DefaultMetaFactory, scheme, scheme, json.SerializerOptions{})
```

Comment:
This should replace with the following right per https://github.com/kubernetes/apimachinery/blob/8d8634786e1cdf4fd147ccf460c043b0168887bd/pkg/runtime/serializer/json/json.go#L46

```suggestion
			yamlSerializer := json.NewSerializerWithOptions(json.DefaultMetaFactory, scheme, scheme, json.SerializerOptions{Yaml: true})
```